common/logger/src/diem_logger.rs

```diff
@@ -165,7 +166,7 @@ impl DiemLoggerBuilder {
         Self {
             channel_size: CHANNEL_SIZE,
             level: Level::Info,
-            remote_level: Level::Debug,
+            remote_level: Level::Info,
```

Comment:
why do we want to change this? I thought the issue was more that we were pulling in logs from crates?

Reply:
Its IMO a saner default, although the expectation is that our production environments would directly configure this via the environment variable so this default wouldn't ever be hit in production.

Reply:
As a principle, I like consistency. Behind the scenes, we want to set this to debug, but in code we want to set it to Info? That doesn't seem sane or consistent. If we look at this in isolation, maybe it is "sane". But I would argue we never were very methodical in setting Debug versus Info :/.